### DISCUSSION TITLE
multipath-ietf-draft-01的xqc_get_pn_ctl函数实现问题

### DISCUSSION THREAD

**Discussion by adcen0107** [alibaba/xquic#242] [Q&A] [answered]
### What happened?

1、multipath-ietf-draft-01分支的 src\transport\xqc_send_ctl.c中xqc_get_pn_ctl函数当前实现如下：
xqc_pn_ctl_t *
xqc_get_pn_ctl(xqc_connection_t *conn, xqc_path_ctx_t *path)
{
    if (conn->enable_multipath == XQC_CONN_MULTIPATH_SINGLE_PNS) {
        return conn->conn_initial_path->path_pn_ctl;
    }

    return path->path_pn_ctl;
}
2、是不是应该修改为如下实现：
    if (conn->enable_multipath == XQC_CONN_NOT_SUPPORT_MULTIPATH) {
        return conn->conn_initial_path->path_pn_ctl;
    }

    return path->path_pn_ctl;

### Steps To Reproduce

检查函数xqc_get_pn_ct实现即可。

**Comment by ruiqizhou** [comment, marked as answer]
https://datatracker.ietf.org/doc/html/draft-ietf-quic-multipath-01#section-9.1
